src/test/java/com/google/javascript/clutz/aliased_enums.d.ts

```diff
@@ -1,14 +1,14 @@
 declare namespace ಠ_ಠ.clutz_internal.nested.bar {
-  type Enum = number ;
-  var Enum : {
+  type HahaEnum = number ;
+  var HahaEnum : {
     A : ಠ_ಠ.clutz_internal.nested.baz.Enum ,
```

Comment:
This is very odd the items are of `Enum` type and not `HahaEnum`, and I think that's why I had the comment "The output is only correct for enums with the same name." before you added fully qualified type output.

However, it does seem to work because they are all `number` after dealiasing, so lets go with it.